packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -34,20 +51,54 @@ const RelationInput = ({
   name,
   label,
   labelLoadMore,
-  listHeight,
   loadingMessage,
-  relations,
   onRelationAdd,
   onRelationLoadMore,
-  onSearchClose,
-  onSearchOpen,
   onRelationRemove,
+  onSearchClose,
   onSearchNextPage,
+  onSearchOpen,
   onSearch,
   placeholder,
   publicationStateTranslations,
+  relations,
   searchResults,
 }) => {
+  const listRef = useRef();
+  const [overflow, setOverflow] = useState('bottom');
```

Comment:
I checked out the branch and played with it a bit: isn't it supposed to work the other way around? By default the last (the item at the top) is cut-off.

Reply:
mmh yes I guess I didn't handle this case yet:
once relations are loaded, we should scroll at the bottom to see the latest relation right? (then you'll see the oldest - at the top - cut off)

Reply:
Yes. I'm ok with adding this in a follow-up if you want